restic/internal/init_test.py

```diff
@@ -48,3 +48,19 @@ def test_init_with_secondary_repository(self, mock_execute):
             'restic', '--json', 'init', '--copy-chunker-params', '--from-repo',
             's3:https://some.backend.com/mybucket'
         ])
+
+    @mock.patch.object(init.command_executor, 'execute')
+    def test_init_with_repository_version(self, mock_execute):
+        mock_execute.return_value = """
+{"message_type":"initialized","id":"d0c84a66bffea61b4cbb88c39cea742127699b3f3af71127b68edcc142edff48","repository":"/tmp/tmp.lNaHYLGR7F"}
+""".strip()
+
+        repository_id = restic.init(repository_version=1)
+
+        self.assertEqual(
+            'd0c84a66bffea61b4cbb88c39cea742127699b3f3af71127b68edcc142edff48',
+            repository_id)
+        mock_execute.assert_called_with([
+            'restic', '--json', 'init', '--repository-version', '1',
+            's3:https://some.backend.com/mybucket'
```

Comment:
This line seems to be accidentally copy/pasted, and it's breaking the test.

Reply:
Now its fixed!